ci/build-scripts/build-backend.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+echo "building backend"
+cd ./backend/
+
+source /opt/rust.env
+
+export DATABASE_URL=postgres://cidb:cidb@127.0.0.1/cidb
+
+# Echo on and fail fast
+set -ex
+
+# Reset Database
+sudo /usr/local/bin/permaplant-reset-ci.sh
```

Comment:
Same here: maybe this can be done outside. It should be also clear at which pipelines it is done (at least write this in a comment).